batcher/aligned-batcher/src/lib.rs

```diff
@@ -285,25 +295,30 @@ impl Batcher {
         if self.user_balance_is_unlocked(addr).await {
             return false;
         }
+        let mut batch_state = self.batch_state.lock().await;
 
-        let mut user_proof_counts = self.user_proof_count_in_batch.lock().await;
-        let user_proofs_in_batch = *user_proof_counts.get(addr).unwrap_or(&0) + 1;
-
+        let user_proofs_in_batch = batch_state
+            .user_proof_count_in_batch
+            .get(addr)
+            .unwrap_or(&0)
+            + 1;
```

Comment:
let's define a method on `BatchState` with this functionality

Reply:
Solved in [2f7ea9d](https://github.com/yetanotherco/aligned_layer/pull/679/commits/2f7ea9dd1cffb4a63d3286eb83b7c96f204daca0)